packages/web-runtime/src/components/UploadInfo.vue

```diff
@@ -169,7 +160,8 @@ export default {
         name: targetRoute.name,
         query: targetRoute.query,
         params: {
-          ...(storageId && path && { storageId })
+          ...(storageId && path && { storageId }),
+          storage: targetRoute.params?.storage
```

Comment:
Is this really always available? Shouldn't it be like the line above so it gets added conditionally?

Reply:
I couldn't find a scenario where this wasn't set. But you're right, better be safe :) Need to rebase and push again anyways.